packages/web/src/js/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const NAME_ARIA_EXPANDED = 'aria-expanded';
```

Comment:
Can we use these constants in all the other web JS as well?

Reply:
Yap, I discussed it with Tomas and I think we agreed that we will put it here one by one.